Expo1900/KoreaItemsTest/KoreaItemsTest.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  KoreaItemsTest.swift
+//  KoreaItemsTest
+//
+//  Created by Brad, Groot on 2022/06/14.
+//
+
+import XCTest
+@testable import Expo1900
+
+class KoreaItemsTest: XCTestCase {
+    var items: NSDataAsset!
+    var jsonDecoder: JSONDecoder!
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        items = NSDataAsset.init(name: "items")
+        jsonDecoder = JSONDecoder()
+    }
+    
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        items = nil
+        jsonDecoder = nil
+    }
+    
+    func test_items파일의_name값이_디코딩되는지() {
+        //given
+        let itemInformation = try? jsonDecoder.decode([KoreaItems].self, from: items.data)
+        
+        //when
+        let result = "직지심체요절"
+        
+        //then
+        XCTAssertEqual(result, itemInformation![0].getName())
```

Comment:
아래와 같이 변경하면 테스트가 동작할까요? 왜 그럴까요?
```suggestion
        XCTAssertEqual(result, itemInformation?[0].getName())
```

Reply:
동작을 확인했습니다.

[91a44f5](https://github.com/yagom-academy/ios-exposition-universelle/pull/173/commits/91a44f505c8e816178afa1ea8ec5e65baeeddce3)

XCTAssertEqual 함수도 `==` 연산자를 사용해서 비교해주기 때문에 ! 를 사용해서 옵셔널을 제거해주는 작업이 불필요하다고 생각이 듭니다.
`아래와 같이 변경하면 테스트가 동작할까요? 왜 그럴까요?`라고 물어보시는 의도를 저희가 파악을 제대로 했나요??

Reply:
`==` 연산자를 통한 비교 과정에서 `result`는 옵셔널이 아니고, `itemInformation?[0].getName()`은 옵셔널 값인데 어떻게 비교가 가능했을까에 대해 여쭈어보려고 했었습니다~! 자세한 내용은 이 문서를 참고해보세요~ [==(_:_:)](https://developer.apple.com/documentation/swift/optional/==(_:_:)-m6x)